addons/project_purchase/models/project.py

```diff
@@ -139,4 +142,26 @@ def _get_profitability_items(self, with_action=True):
                 costs['data'].append(purchase_order_costs)
                 costs['total']['billed'] += amount_invoiced
                 costs['total']['to_bill'] += amount_to_invoice
+            # calculate the cost of bills without a purchase order
+            bills_move_line_read = self.env['account.move.line'].sudo().search_read([
+                ('analytic_account_id', 'in', self.analytic_account_id.ids),
+                ('move_id.move_type', 'in', ['in_invoice']),
+                ('parent_state', '=', 'posted'),
+                ('price_subtotal', '>', 0),
+                ('id', 'not in', purchase_order_line_invoice_line_ids)
+            ], ['price_subtotal'])
+            if bills_move_line_read:
+                amount_invoiced = 0.0
+                for moves_read in bills_move_line_read:
+                    amount_invoiced -= moves_read['price_subtotal'] if moves_read['price_subtotal'] > 0 else 0.0
```

Comment:
The condition `moves_read['price_subtotal'] > 0` is not necessary since we should only have account move line with `price_subtotal` > 0. 🙂 
```suggestion
                    amount_invoiced -= moves_read['price_subtotal']
```

What do you think? 🙂